<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SeerEditorWidgetAssemblyForm</class>
 <widget class="QWidget" name="SeerEditorWidgetAssemblyForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>833</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SeerEditorWidgetAssemblyForm</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="SeerEditorWidgetAssemblyArea" name="assemblyWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>100</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="searchBarLayout">
     <item>
      <widget class="QClearLineEdit" name="searchTextLineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>50</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Enter text to search for.</string>
       </property>
       <property name="placeholderText">
        <string>Enter search text</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="matchCaseCheckBox">
       <property name="text">
        <string>Match Case</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="matchesLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="searchDownToolButton">
       <property name="toolTip">
        <string>Search for the next occurance.</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/seer/resources/RelaxLightIcons/go-down.svg</normaloff>:/seer/resources/RelaxLightIcons/go-down.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="searchUpToolButton">
       <property name="toolTip">
        <string>Search for the previous occurance.</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/seer/resources/RelaxLightIcons/go-up.svg</normaloff>:/seer/resources/RelaxLightIcons/go-up.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_1">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="searchLineNumberLineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>50</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Scroll to a specific 0xAddress, +Offset, or line#.</string>
       </property>
       <property name="placeholderText">
        <string>Go to a 0xAddress, +Offset, or line#.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="searchCloseToolButton">
       <property name="toolTip">
        <string>Close the search bar.</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/seer/resources/RelaxLightIcons/list-remove.svg</normaloff>:/seer/resources/RelaxLightIcons/list-remove.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SeerEditorWidgetAssemblyArea</class>
   <extends>QWidget</extends>
   <header location="global">SeerEditorWidgetAssembly.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QClearLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">QClearLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
